packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -50,6 +50,12 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   beakWidth?: number;
 
   /**
+   * Custom width for callout. If value is 0, no width is applied.
+   * @default 0
+   */
+  calloutWidth?: number;
```

Comment:
You might want to make sure that the user know that this width _includes_ the border, as though it were ` box-sizing: border-box `